File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/errors/OSCValidationError.java
package ac.at.fhstp.digitech.oscexchange.errors;

import ac.at.fhstp.digitech.oscexchange.OSCArgs;

/**
 * An error that occurs when OSCArgs fail to be validated
 */
public final class OSCValidationError extends OSCArgsError {

    public OSCValidationError(Exception exception, OSCArgs args) {
        super(exception, args);
    }

}

File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/errors/OSCValidationError.kt
package ac.at.fhstp.digitech.oscexchange.errors

import ac.at.fhstp.digitech.oscexchange.OSCArgs

/**
 * An error that occurs when OSCArgs fail to be validated
 */
class OSCValidationError(exception: Exception?, args: OSCArgs) :
    OSCArgsError(exception, args)